README.md

```diff
@@ -26,11 +26,11 @@ include mosquitto
 All parameters are documented in the REFERENCE.md. You can pass configuration options with:
 
 ```puppet
-class{'mosquitto:
+class { 'mosquitto:
```

Comment:
Only now I spot the syntax error that was also already present: missing a closing quote.